samples/ChatbotExamplev2/Bots/PersonalizerChatbot.cs

```diff
@@ -244,6 +258,14 @@ private async Task RewardAsync(ITurnContext turnContext, string eventId, double
             await _personalizerClient.RewardAsync(eventId, new RewardRequest(reward), cancellationToken);
```

Comment:
Please add comment that highlights this is the other of the 2 calls to Personalizer.

Reply:
added comment